conf/messages

```diff
@@ -544,3 +527,24 @@ month.12 = December
 # -----------------------------------------------------------
 otherReason.uploadEvidence.removeFile.headingAndTitle = Are you sure you want to remove file {0}?
 otherReason.uploadEvidence.removeFile.error.required = Select yes if you want to remove this file
+
+# You Cannot Appeal page
+# -----------------------------------------------------------
+youCannotAppeal.headingAndTitle.lpp = You cannot use this service until you pay your VAT
+agent.youCannotAppeal.headingAndTitle.lpp = You cannot use this service until your client pays the VAT
+youCannotAppeal.headingAndTitle.lsp = You cannot use this service until you submit your VAT Return
+agent.youCannotAppeal.headingAndTitle.lsp = You cannot use this service until the VAT Return is submitted
+youCannotAppeal.p1 = You can still write to HMRC and ask us to review the penalty. Details are on the penalty notice we sent you through the post.
+youCannotAppeal.p2.lpp = If you want to use this service to appeal, you need to pay your VAT first.
+agent.youCannotAppeal.p2.lpp = If you want to use this service to appeal, your client needs to pay their VAT first.
+youCannotAppeal.p2.lsp = If you want to use this service to appeal, you need to submit your VAT Return first.
+agent.youCannotAppeal.p2.lsp = If you want to use this service to appeal, you or your client need to submit the VAT Return first.
+youCannotAppeal.p3.lpp = It can take up to 5 days for the payment to clear and show on your payment history. If you‘ve already paid, keep checking back to see when the payment clears.
+agent.youCannotAppeal.p3.lpp = It can take up to 5 days for the payment to clear and show on their payment history. If they‘ve already paid, keep checking back to see when the payment clears.
+youCannotAppeal.p3.lsp = It can take up to 24 hours before the VAT Return shows on the VAT account. If you‘ve already submitted it, keep checking back to see when it appears in your submission history.
```

Comment:
```suggestion
youCannotAppeal.p3.lsp = It can take up to 24 hours before the VAT Return shows on your VAT account. If you‘ve already submitted it, keep checking back to see when it appears in your submission history.
```